react/modules/seller.ts

```diff
@@ -1,13 +1,15 @@
 import { ProductTypes } from 'vtex.product-context'
 
-export function getFirstAvailableSeller(sellers?: ProductTypes.Seller[]) {
+export function getDefaultSeller(sellers?: ProductTypes.Seller[]) {
   if (!sellers || sellers.length === 0) {
     return
   }
 
-  const availableSeller = sellers.find(
-    seller => seller.commertialOffer.AvailableQuantity !== 0
-  )
+  const defaultSeller = sellers.find(seller => seller.sellerDefault)
 
-  return availableSeller
+  if (!defaultSeller) {
+    return sellers[0]
+  }
```

Comment:
Why not keep the same selection logic used before in this case?

```tsx
sellers.find(
    seller => seller.commertialOffer.AvailableQuantity !== 0
  )
```

Reply:
Because all other repos are selecting the first. This would be the only component using the first available seller.

Reply:
I left a comment regarding this:

> Also tested in motorolauk
> https://brenovtex--motorolauk.myvtex.com/
> So we don't have any regression to the behavior that was fixed at #52

Reply:
😂  Oookay